uint8 goal_elevator_position